src/components/analyses/landing/AnalysisSubmissionLanding.js

```diff
@@ -392,28 +393,54 @@ export default function AnalysisSubmissionLanding(props) {
                                             {!isSmDown && <>{t("refresh")}</>}
                                         </Button>
                                     </Grid>
-                                    <Grid item>
-                                        {allowShareWithSupport && (
+                                    {allowTimeExtn && (
+                                        <Grid item>
                                             <Button
                                                 id={buildID(
                                                     baseId,
-                                                    ids.SHARE_WITH_SUPPORT
+                                                    ids.MENUITEM_EXTEND_TIME_LIMIT
                                                 )}
                                                 variant="outlined"
-                                                onClick={() =>
-                                                    setHelpOpen(true)
-                                                }
                                                 size="small"
+                                                disableElevation
+                                                color="primary"
+                                                onClick={() => {
+                                                    setConfirmExtendTimeLimitDlgOpen(
+                                                        true
+                                                    );
+                                                }}
+                                                startIcon={<HourGlassIcon />}
+                                            >
+                                                {!isSmDown && (
+                                                    <>{t("extendTime")}</>
```

Comment:
This Fragment looks unnecessary, since `{!isSmDown && t("extendTime")}` should work the same.

I see it's already used like this in other places in this file, so it's fine to leave it like this for now just for consistency in this file.

Reply:
Ha, yup, I just copied the others in the file assuming it was necessary. I'll leave it for now, but I'll keep it in mind as I make more UI changes (which it looks like I will be).